sympy/assumptions/ask.py

```diff
@@ -134,7 +135,11 @@ def ask(proposition, assumptions=True, context=global_assumptions):
 
     local_facts = _extract_facts(assumptions, expr)
 
-    if local_facts and satisfiable(And(local_facts, known_facts_cnf)) is False:
+    known_facts_cnf = get_known_facts_cnf()
+    known_facts_dict = get_known_facts_dict()
+
+    if local_facts \
+    and satisfiable(And(local_facts, known_facts_cnf)) is False:
```

Comment:
I'd prefer if this was line broken as:
Why did you line break this? With out the linebreak, it's still only 79 characters long...

If you must do a line break, please indent two levels to indicate that it's for formatting (as per the pep); having it be on the same level as the if just looks weird. I'd prefer to break it in the parens, and ditch the backslash entirely, but that's just me.